Tests/Rules/PlaceOpenBrace.tests.ps1

```diff
@@ -33,6 +33,37 @@ function foo ($param1)
         It "Should mark only the open brace" {
             $violations[0].Extent.Text | Should -Be '{'
         }
+
+        It "Should correct violation" {
+            $scriptDefinition = @'
+foreach ($x in $y)
+{
+    Get-Something
+}
+'@
+            $expected = @'
+foreach ($x in $y) {
+    Get-Something
+}
+'@
+            Invoke-Formatter -ScriptDefinition $scriptDefinition -Settings $settings | Should -Be $expected
+        }
+
+        It "Should correct violation and take comment into account" {
+            $scriptDefinition = @'
+foreach ($x in $y) # useful comment
+{
+    Get-Something
+}
+'@
+            $expected = @'
+foreach ($x in $y) { # useful comment
+    Get-Something
+}
+'@
+            Invoke-Formatter -ScriptDefinition $scriptDefinition -Settings $settings | Should -Be $expected
+        }
+
```

Comment:
Is it possible to have the reverse of this?

We have:
```
foreach ($x in $y) { # PSSA is awesome
    Get-Something
}
```

and want:
```
foreach ($x in $y) # PSSA is awesome
{
    Get-Something
}
```

would it be worth having a test for that?

Reply:
This is a very good point. We should test this case as well, I have not thought of that.

Reply:
I added a test case for this now and added implicit integration tests with the code formatting style presets that VSCode uses.